docs/politics.md

```diff
@@ -95,37 +95,37 @@ All policies instituted by the government are going to have pros and cons. Let's
 
-Also see [the section on voting](#voting).
+Also, see [the section on voting](#voting).
 
 ## Social Justice
 
-I think that social justice is a real thing. We have made a ton of progress so far (in e.g. ending slavery, ending redlining, legalizing gay marriage) but we are not at the finish line yet. In general, the government should work to protect the rights of minority groups, including trans people.
+I believe that social justice is a real thing. We have made a ton of progress so far (e.g. ending slavery, ending redlining, legalizing gay marriage) but we are not at the finish line yet. In general, the government should work to protect the rights of minority groups, including trans people.
 
-Sometimes, social justice can go too far (e.g. [Rachel Dolezal](https://en.wikipedia.org/wiki/Rachel_Dolezal) & "transracialism," Twitch banning the "blind playthrough" tag, pronouns based on animals). On the other hand, a lot of people against social justice have ridiculous and overblown concepts of what social justice even is (from e.g. watching "SJW owns" videos on YouTube).
+Sometimes, social justice can go too far (e.g. [Rachel Dolezal](https://en.wikipedia.org/wiki/Rachel_Dolezal) & "transracialism," Twitch banning the "blind playthrough" tag, pronouns based on animals). On the other hand, a lot of people against social justice have ridiculous and overblown concepts of what social justice even is (e.g. from watching "SJW owns" videos on YouTube).
 
 The right place is somewhere in the middle. I advocate that you should try to have respect for other people, especially if doing so won't inconvenience you that much. In general, this means that if someone wants you to call them she/her, then you should probably be polite and do so.
 
-Also see the section on [systemic racism](#systemic-racism).
+Also, see the section on [systemic racism](#systemic-racism).
 
 ## Diversity
 
 In general, I believe diversity is a good thing, though there are times when it can be taken too far. It would be a mistake for groups and organizations to strive for a purely equal representation of all genders and races — the best person should get the job, regardless of what gender or race they are.
 
-At the same time, though, there is a big difference between a group of 10 men versus a group of 9 men + 1 women. When hiring the 10th person for this group, if a man and a women have similar credentials, it might benefit the group a lot more to hire the women — the group would gain a different perspective and it would be more well-rounded.
+At the same time, though, there is a big difference between a group of 10 men versus a group of 9 men + 1 woman. When hiring the 10th person for this group, if a man and women have similar credentials, it might benefit the group a lot more to hire the women — the group would gain a different perspective and it would be more well-rounded.
```

Comment:
```suggestion
At the same time, though, there is a big difference between a group of 10 men versus a group of 9 men + 1 woman. When hiring the 10th person for this group, if a man and woman have similar credentials, it might benefit the group a lot more to hire the woman — the group would gain a different perspective and it would be more well-rounded.
```
If the singular woman is spelled with an a, the other instances in the same paragraph should be updated as well.